### PR TITLE
Remove support for parallel Java8 streams.

### PR DESCRIPTION
**Pull request by chrisr3**
Any request for a parallel stream will return a sequential stream instead. (The specification permits this.)

### CODE REVIEW

File: jdk/src/share/classes/java/util/stream/SliceOps.java [hunk 1]
```diff
@@ -160,29 +160,7 @@ private static long calcSliceFence(long skip, long limit) {
             <P_IN> Node<T> opEvaluateParallel(PipelineHelper<T> helper,
```
Comment (exFalso) [inline jdk/src/share/classes/java/util/stream/SliceOps.java]: Should we remove these functions instead of throwing?

**Review by exFalso — COMMENTED** [review]
One thing

File: jdk/src/share/classes/java/util/stream/SliceOps.java (hunk 1 above)
Comment (chrisr3) [reply jdk/src/share/classes/java/util/stream/SliceOps.java]: Well, I _could_ remove them... (probably...). But I think I was concerned about deleting methods from public Java interfaces.

Comment (exFalso) [reply jdk/src/share/classes/java/util/stream/SliceOps.java]: Isn't that better in general than throwing an exception? If someone's developing a contract we probably don't want to have this exposed on the public api

**Review by exFalso — APPROVED** [review]
LGTM 👍

**Pull request merged**